Apps/DebugApp/DebugApp.xcodeproj/xcshareddata/xcschemes/DebugApp.xcscheme

```diff
@@ -41,8 +41,12 @@
             isEnabled = "YES">
          </AdditionalOption>
       </AdditionalOptions>
-      <Testables>
-      </Testables>
+      <TestPlans>
+         <TestPlanReference
+            reference = "container:../../TestPlans/DebugApp.xctestplan"
```

Comment:
DebupApp testplan is empty. Lets replace it with UnitTests plan

Reply:
👍  -- updated in most recent commit